android/KMEA/app/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,13 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.tavultesoft.kmea" >
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  package="com.tavultesoft.kmea" >
 
-    <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Do we need this perm in Engine?

Reply:
So we should be able to remove this in Engine and add it in apps instead?

Reply:
Right. l.9 `tools:node="remove"` removes the INTERNET permission from the manifest (even if a dependency like sentry tries to add it back).

Then the KMAPro Manifest l.7 uses `tools:node="replace"` to ensure the INTERNET permission is "added".

Reply:
But I'm looking at `    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />`, not `INTERNET`?

Reply:
I'll put this PR back in draft to also remove ACCESS_NETWORK_STATE permission

Reply:
@keyman-test-bot retest TEST_KMSAMPLE1_SDK_30